upgrade.md

```diff
@@ -79,6 +79,22 @@ The constructor signature of the `Illuminate\Auth\Access\Response` class has cha
 
 The `Illuminate\Contracts\Auth\Access\Gate` contract has received a new `inspect` method. If you are implementing this interface manually, you should add this method to your implementation.
 
+#### The `Illuminate\Auth\Access\HandlesAuthorization` Trait
+
+**Likelihood Of Impact: Low**
+
+The `Illuminate\Auth\Access\HandlesAuthorization` trait's method `deny()` has been update to return `\Illuminate\Auth\Access\Response`. Previously it used to throw an exception. 
+If you have been using this method in your policy classes to have a `return` statement like:
```

Comment:
This sentence seems to be missing a word or two. Maybe the word "add"?
Perhaps instead of starting with "If" maybe it should just be "You now need to ..." and then be clear about what's needed.

Reply:
please check latest commit